### PR TITLE
remove hardcoded colors add yarn.lock to .gitignore

### PR DESCRIPTION
**Pull request by JasonLantz**

### CODE REVIEW

**Comment by StommePoes** [comment]
look ok

**Comment by wyseguyonline** [comment]
lgtm

**Comment by StommePoes** [comment]
is this waiting for Eajaz?

**Comment by umahaea** [comment]
yes Mallory..

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ _gh_pages
 .idea
 test/fixtures/color/*.json
 test.json
-elements.iml
\ No newline at end of file
+elements.iml
+yarn.lock
```
Comment (StommePoes) [inline .gitignore]: No idea, but there were apparently some loose cats in the repo and they needed to be distracted by the new shiny.

Comment (wyseguyonline) [inline .gitignore]: Very well played Mallory!

**Pull request merged**